packages/logrotate/build

```diff
@@ -21,9 +21,9 @@ nomail
 
 /var/lib/dcos/mesos/log/* {
     olddir /var/lib/dcos/mesos/log/archive
-    maxsize 2000k
-    daily
-    rotate 7
+    maxsize 256M
```

Comment:
Keep in mind that `maxsize` is the size of a single file, not the whole directory.

How large is the total log size allowed be be?  10 GB?
As long as we get rid of the GLOG logrotate, we can keep the `rotate N` option here and have no disk leakage.